src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java

```diff
@@ -71,50 +71,94 @@ public void init(FMLInitializationEvent event) {
 		}
 
 		// Cooking for Blockheads II
-		if(CookingConfig.enableCraftingBook) {
-			GameRegistry.addRecipe(new ItemStack(CookingForBlockheads.itemRecipeBook, 1, 1), " C ", "DBD", " C ", 'C', Blocks.crafting_table, 'D', Items.diamond, 'B', CookingForBlockheads.itemRecipeBook);
+		if (CookingConfig.enableCraftingBook) {
+			if (Loader.isModLoaded("gregtech")) {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.itemRecipeBook, 1, 1), " C ", "DBD", " C ", 'C', Blocks.crafting_table, 'D', "plateSteel", 'B', CookingForBlockheads.itemRecipeBook));
```

Comment:
I thought it was supposed to be pre-steam? Steel is in the middle of steam.

Reply:
why it must be avaialable pre steam ? Yea have much todo with other stuff. Planting stuff is always steam age in my playthrough

Reply:
I want it early game.  Steam age is fine, or some pieces LV.